class.photon.php

```diff
@@ -60,6 +60,9 @@ private function setup() {
 
 		// Responsive image srcset substitution
 		add_filter( 'wp_calculate_image_srcset', array( $this, 'filter_srcset_array' ), 10, 4 );
+		if ( ! has_filter( 'wp_calculate_image_sizes') ) { // To allow themes to fully control the sizes array.
```

Comment:
I'm open to it. I don't have strong opinions on how we do it, as long as we let theme authors be smarter than we can be globally. I'm trying to think what kind of use case would modifying it before a theme gets it would cause problems; nothing yet.